apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -209,9 +209,10 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
    * @return {SerialPort}
    */
   static openSerialPort(portName) {
-    return new ChromeSerialPort.SerialPort(portName, {
-      bitrate: SERIAL_BAUD
-    }, true);
+    return new SerialPort(portName, {
+      autoOpen: true,
+      bitrate: SERIAL_BAUD,
+    });
```

Comment:
The API change here is because [`browser-serialport`](https://github.com/garrows/browser-serialport) (and by proxy [`chrome-serialport`](https://github.com/bcjordan/chrome-serialport)) are modeled on an older version of [`node-serialport`](https://github.com/EmergingTechnologyAdvisors/node-serialport).  For the bridge app I'm using the latest version of `node-serialport` (4.0.7), and the `openImmediately` boolean argument became the `autoOpen` option [in v4.0.0](https://github.com/EmergingTechnologyAdvisors/node-serialport/releases/tag/4.0.0).